source/operations/cert-manager/cert-manager-operator.rst

```diff
@@ -0,0 +1,239 @@
+------------------------------------------
+
+Use the secret created above to add an ``Issuer`` resource for the ``minio-operator`` namespace.
+
+1. Create a file called ``operator-ca-issuer.yaml`` with the following contents: 
+
+   .. code-block:: yaml
+
+      # operator-ca-issuer.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Issuer
+      metadata:
+        name: minio-operator-ca-issuer
+        namespace: minio-operator
+      spec:
+        ca:
+          secretName: operator-ca-tls
+
+
+2. Apply the resource to the cluster
+   
+   .. code-block:: shell
+
+      kubectl apply -f operator-ca-issuer.yaml
+
+3) Create TLS certificate
+-------------------------
+
+Now that the ``Issuer`` exists in the ``minio-operator`` namespace, cert-manager can add a certificate.
+
+The certificate from cert-manager must be valid for the following DNS domains:
+
+- ``sts``
+- ``sts.minio-operator.svc.``
+- ``sts.minio-operator.svc.<cluster domain>``
+
+  .. important::
+
+      Replace ``<cluster domain>`` with the actual value for your MinIO tenant.
+      ``cluster domain`` is the internal root DNS domain assigned in your Kubernetes cluster. 
+      Typically, this is ``cluster.local``, but confirm the value by checking your coredns configuration for the correct value for your Kubernetes cluster. 
+      
+      For example:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl get configmap coredns -n kube-system -o jsonpath="{.data}"
+
+      Different Kubernetes providers manage the root domain differently.
+      Check with your Kubernetes provider for more information.
+
+1. Create a ``Certificate`` for the domains mentioned above:
+
+   Create a file named ``sts-tls-certificate.yaml`` with the following contents:
+
+   .. code-block:: yaml
+      :class: copyable
+      :emphasize-lines: 7,12
+
+      # sts-tls-certificate.yaml
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: sts-certmanager-cert
+        namespace: minio-operator
+      spec:
+        dnsNames:
+          - sts
+          - sts.minio-operator.svc
+          - sts.minio-operator.svc.cluster.local # Replace cluster.local with the value for your domain.
+        secretName: sts-tls
+        issuerRef:
+          name: minio-operator-ca-issuer
+
+   .. important::
+   
+      The ``spec.secretName`` is not optional.
+   
+      The secret name **must** be ``sts-tls``.
+      Confirm this by setting ``spec.secretName: sts-tls`` as highlighted above.
+
+2. Apply the resource to the cluster:
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f sts-tls-certificate.yaml
+
+This creates a secret called ``sts-tls`` in the ``minio-operator`` namespace.
+
+.. warning::
+  
+   Failing to provide the ``sts-tls`` secret containing the TLS certificate or providing an invalid ``key-value`` pair in the secret will prevent the STS service from starting.
```

Comment:
A possible version in case this warning should be more alarming
```suggestion
   The STS service will not start if the ``sts-tls`` secret, containing the TLS certificate, is missing or contains an invalid ``key-value`` pair. 
```

Reply:
I'll take that suggestion.